pkgs/development/libraries/wlroots/0.15.nix

```diff
@@ -22,15 +22,14 @@ stdenv.mkDerivation rec {
   # $out for the library and $examples for the example programs (in examples):
   outputs = [ "out" "examples" ];
 
-  strictDeps = true;
```

Comment:
Likely an unintended change?
(It's still in 0.14.nix and you did add it in https://github.com/NixOS/nixpkgs/pull/172186.)

Other than that I didn't notice anything strange :)

Reply:
yep, wonder how it got there.

Reply:
ah it got there because i pressed u too many times and undid the change